apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -270,6 +270,19 @@ function initializeBlocklyWrapper(blocklyInstance) {
     CdoRendererZelos,
     true /* opt_allowOverrides */
   );
+  // cleanUp() doesn't currently account for immovable blocks.
+  blocklyWrapper.blockly_.ContextMenuRegistry.registry.unregister(
```

Comment:
Nit: might make sense organizationally to put these in [contextMenu.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/blockly/addons/contextMenu.js) and add a function like unregisterContextMenuItems()?

Reply:
Good suggestion. Done!